test/e2e/components/vl-input-field.js

```diff
@@ -1,10 +1,11 @@
 const { VlElement } = require('vl-ui-core').Test;
 const { VlFormValidation } = require('vl-ui-form-validation').Test;
+const { Key } = require('selenium-webdriver');
 
 class VlInputField extends VlFormValidation { 
     
     async setValue(content) {
-    	await this.clear();
+    	await super.clear();
```

Comment:
super.clear() en dan overschrijft ge de clear, dus dat kan even goed this zijn denk ik

Reply:
ik weet niet... de super.clear() triggert geen events... althans de input-event niet (welke getest is).

Reply:
hangt er wat vanaf hoe je het bekijkt... ik wil het gerust aanpassen hoor

Reply:
ik begrijp niet goed hoe het er van af zou hangen hoe je het bekijkt. als je super.clear() doet maar dan zelf clear() overschrijft is dat toch hetzelfde dan this.clear()

Reply:
Bij setValue wil je geen event sturen op de clear, dus wordt de super gebruikt. In dit geval super.clear != this.clear, dus lijkt me de implementatie goed.

Reply:
Wellicht begrijp ik het niet goed dan, maar de super.clear is toch de clear van WebElement?
En de this.clear() die de clear die net zelf geschreven is?

Reply:
ja sorry t was een briefke